src/nvim/option.c

```diff
@@ -454,6 +454,9 @@ void set_init_1(void)
   /* Set all options to their Vim default */
   set_options_default(OPT_FREE);
 
+  // set 'laststatus'
+  last_status(false);
```

Comment:
@bfredl I think that this is clearly a `set_options_default` bug which should be fixed in `set_options_default` body and not worked around here.